kani-driver/src/call_single_file.rs

```diff
@@ -117,6 +117,9 @@ impl KaniSession {
 
         if self.args.debug {
             flags.push("--log-level=debug".into());
+        } else if self.args.verbose {
+            // Print the symtab command being invoked.
+            flags.push("--log-level=info".into());
```

Comment:
This was removed in https://github.com/model-checking/kani/pull/1730 is the underlying issue fixed there?